filebeat/input/log/harvester.go

```diff
@@ -306,7 +306,7 @@ func (h *Harvester) Run() error {
 		}
 	}(h.state.Source)
 
-	logger.Info("Harvester started for file.")
+	logger.Infof("Harvester started for file: %v", h.config.Paths)
 
```

Comment:
The log statement becomes misleading, the value of `h.config.Path` is not an exact file, it can be a path with a wildcard.
```suggestion
	logger.Infof("Harvester started for paths: %v", h.config.Paths)
```

Reply:
I believe what you are doing is the right thing, note that the execution of the harvester is a bit confused and lost in my memories. The code managing the reading of the files in the old log input is not obvious at all.